frontend/src/Album/Details/AlbumDetails.js

```diff
@@ -301,7 +301,7 @@ class AlbumDetails extends Component {
               <AlbumCover
                 className={styles.cover}
                 images={images}
-                size={500}
+                size={250}
```

Comment:
Make same change to ArtistDetails.js, we should use 250 image given the container is 250x250px